t/FHEM/14_SD_UT/testData.json

```diff
@@ -602,1578 +521,1370 @@
-            "minProtocolVersion" : "1.27",
-            "user" : "SD_Protocol",
+            "comment" : "(ID 8,42,63 additionally)",
+            "dispatch_repeats" : "1",
+            "dmsg" : "P69#00E6BE9107C",
+            "internals" : {
+               "DEF" : "HSM4 E6BE910",
+               "NAME" : "HSM4_E6BE910"
+            },
             "readings" : {
                "deviceCode" : "1110011010111110100100010000",
                "state" : "button_1"
             },
+            "rmsg" : "MU;P0=-508;P1=1029;P2=503;P3=-1023;P4=12388;D=01010232323232310104010101010101010102323231010232310231023232323231023101023101010231010101010232323232310104010101010101010102323231010232310231023232323231023101023101010231010101010232323232310104010101010101010102323231010232310231023232323231023101;CP=2;R=37;O;",
             "tests" : [
                {
-                  "comment" : "#0",
                   "attributes" : {
                      "model" : "HSM4"
-                  }
+                  },
+                  "comment" : "#0"
                }
-            ],
-            "internals" : {
-               "DEF" : "HSM4 E6BE910",
-               "NAME" : "HSM4_E6BE910"
-            },
-            "comment" : "(ID 8,42,63 additionally)",
-            "revision_entry" : "2021-04-22 22:03:10",
-            "dmsg" : "P69#00E6BE9107C",
-            "dispatch_repeats" : "1"
+            ]
          }
-      ]
+      ],
+      "id" : "69",
+      "module" : "SD_UT",
+      "name" : "HSM4"
    },
    {
-      "name" : "LED XM21",
-      "module" : "SD_UT",
-      "id" : "76",
       "data" : [
          {
+            "dmsg" : "P76#FFFFFFFFFFFFFFFF",
             "internals" : {
                "DEF" : "LED_XM21_0 FFFFFFFFFFFFFF",
                "NAME" : "LED_XM21_0_FFFFFFFFFFFFFF"
             },
+            "readings" : {
+               "deviceCode" : "11111111111111111111111111111111111111111111111111111111",
+               "state" : "on"
+            },
+            "rmsg" : "MU;P0=-205;P1=113;P3=406;D=010101010101010101010101010101010101010101010101010101010101030303030101010101010101010101010101010101010101010101010101010101010101010101010101010101010101010101010101010101010101010101010101010103030303010101010101010101010100;CP=1;R=69;",
             "tests" : [
                {
                   "attributes" : {
                      "model" : "LED_XM21_0"
                   },
                   "comment" : "#0"
                }
-            ],
+            ]
+         },
+         {
+            "dmsg" : "P76#FFFFFFFFFFFFFFC",
+            "internals" : {
+               "DEF" : "LED_XM21_0 FFFFFFFFFFFFFF",
+               "NAME" : "LED_XM21_0_FFFFFFFFFFFFFF"
+            },
             "readings" : {
                "deviceCode" : "11111111111111111111111111111111111111111111111111111111",
-               "state" : "on"
+               "state" : "off"
```

Comment:
Da bin ich jetzt  überrascht, dass hier ein anderer Wert im Reading steht.

Reply:
@sidey79 schau dir das Ganze richtig an. Du siehst eine "optische Täuschung" das der State sich änderte.
Das ist für dich so sichtbar in der Oberfläche weil 1. Zeilen entfernt wurden | 2. die Sortierung sich änderte der Einträge.

Hier https://github.com/RFD-FHEM/RFFHEM/blob/master/t/FHEM/14_SD_UT/testData.json#L882:L899 siehst du den ursprünglichen Eintrag und hier https://github.com/RFD-FHEM/RFFHEM/blob/master_t_rev/t/FHEM/14_SD_UT/testData.json#L764:L783 die Zeilen im neuen Branch.
Reading "on" ist jeweils im comment #0.